addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -16,31 +16,42 @@ Note that the commands are executed in the context and with the privileges of th
 
 The `command` Thing requires the command to execute on the shell. Optionally one can specify:
 
-- a transformation to apply on the execution result, 
-- an interval, in seconds, the command will be repeatedly executed, 
-- a time-out, in seconds, the execution of the command will time out, and lastly, 
-- a boolean parameter to make the command execute immediately every time the state of the input channel has changed. 
+- [transform] a transformation to apply on the execution result, 
+- [interval] an interval, in seconds, the command will be repeatedly executed, 
+- [timeput] a time-out, in seconds, the execution of the command will time out, and lastly, 
+- [runOnInput] a boolean parameter to make the command execute immediately every time the state of an input channel is set by a Command 
```

Comment:
It appears that runOnInput (I actually like 'autorun' better, but as a channel-level config parameter) makes "command execute immediately every time a command is received on the input channel". The command does not necessarily set the state of the item, even indirectly, if autoupdate=false. In general, the runOnInput operates independently of of the item state.